### ISSUE TITLE
EnMAP-Box News Panel

### ISSUE THREAD

**Issue by jakimowb** [EnMAP-Box/enmap-box#841]
Add a news panel that allows to display EnMAP-Box News, similar to the QGIS News panel.
![grafik](https://github.com/EnMAP-Box/enmap-box/assets/1404870/92762212-3787-45f7-9da7-c7146678e545)

We like to use it to inform EnMAP-Box users on workshops or the upcomming user survey.

The EnMAP-Box News Panel should:
* [ ] download the news from a remote server 
* [ ] do not show any error in case the remote server is not available
* [ ] evaluate date ranges which are specified on the remote server, e.g.  to not display outdated news and upcomming events only. If no relevant news are available, the entire news panel should not be displayed at all
* [ ] be able to be skipped / set hidden

**Comment by janzandr** [comment]
> download the news from a remote server

@jakimowb what about just storing a **news.json** file inside the code repo? E.g.
```
# https://raw.githubusercontent.com/EnMAP-Box/enmap-box/main/news.json:
[
    {
        "title": "Workshop 2025",
        "date_end": "'2025-05-01",
        "message": "The 42th EnMAP User Workshop will take place..."
    }
    ....
]
```

**Comment by janzandr** [comment]
Let's also have an image item:
```
[
  {
    "title": "News 1",
    "image": "https://www.enmap.org/_nuxt/img/9ebc66d.png",
    "date_end": "2025-05-01",
    "message": "Message 1"
  },
  {
    "title": "News 2",
    "image": "https://www.enmap.org/_nuxt/img/9ebc66d.png",
    "date_end": "2025-05-01",
    "message": "Message 2"
  },
  {
    "title": "Outdated news",
    "image": "https://www.enmap.org/_nuxt/img/9ebc66d.png",
    "date_end": "2020-05-01",
    "message": "Message 3"
  }
]
```

**Comment by jakimowb** [comment]
Probably it's best to inspect https://github.com/qgis/qgis-feed t see how it is done in QGIS and use a reduced approach.
The feed json (or what ever is used as endpoint) could be hosted on the EnMAP-Box documentation page or a separated repo.

**Comment by janzandr** [comment]
> be able to be skipped / set hidden

This should be handled via an EnMAP-Box setting, which is not yet implemented, see #72 for details.

**Comment by janzandr** [comment]
Added a new setting:
![image](https://github.com/EnMAP-Box/enmap-box/assets/45030715/0a185f9a-5fd0-4980-bb18-672d4c2a40c5)

**Comment by janzandr** [comment]
Like in QGIS news, we should also have a weblink for each news item.

**Comment by jakimowb** [comment]
Would be nice to have something that updates the news in https://enmap-box.readthedocs.io/en/latest/index.html as well.
Just an idea: in enmap-box-documentation we could create a "news" folder that contains *.rst or *.md file, one for each news item.
From that the required *.json is generated, as well the *.rst file content that is shown in https://enmap-box.readthedocs.io/en/latest/index.html#

**Comment by jakimowb** [comment]
Indeed, parsing the source/news.rst and generating the news json from instead is probably much simpler. 
For that the news.rst entries may contain some special tag in a sphinx comment, like

```rst

Example rst news section
===============
.. 
  # news feed tags
  date_end = 2024-05-05
  date_begin = 2024-01-01

This is an RST Style news entry to be shown at the `documentation page <https://enmap-box.readthedocs.io/en/latest/index.html#news>`_
```
